template/sync_template.py

```diff
@@ -0,0 +1,68 @@
+"""This script serves as a skeleton template for synchronous AgentQL scripts."""
+
+import logging
+
+import agentql
+from agentql.ext.playwright.sync_api import Page
+from playwright.sync_api import sync_playwright
+
+# Set up logging
+logging.basicConfig(level=logging.INFO)
+log = logging.getLogger(__name__)
+
+# Set the URL to the desired website
+URL = "WEBSITE_URL"
+
+# Update the query to locate the desired element on the page
+ELEMENTS_QUERY = """
+{
+    search_input
+    search_btn
+}
+"""
+
+# Update the query to fetch the desired data from the page
+DATA_QUERY = """
+{
+    products[] {
+        price
+    }
+}
+"""
+
+
+def main():
+    with sync_playwright() as playwright, playwright.chromium.launch(headless=True) as browser:
```

Comment:
feels like we can be less verbose here, since it's local to the with-header, and the whole line is long
```suggestion
    with sync_playwright() as p, p.chromium.launch(headless=True) as browser:
```